src/sections/Home/styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const VideoHome = styled.video`
+    width: 100vw;
+    opacity: 0.6;
+    padding-top: ${({theme})=> theme.scale.base*6.5}px;
+    position: absolute;
+    z-index: -1;
+`;
+
+export const HomeText = styled.p`
```

Comment:
the text is too huge for mobile devices, maybe needs an absolute positioning and overeflow hidden on the parent so that it doesnt overlap the rest of the content